packages/terra-form-input/src/Input.jsx

```diff
@@ -19,6 +19,10 @@ const propTypes = {
    * Whether the input is disabled.
    */
   disabled: PropTypes.bool,
+  /**
+   * Whether the field is incomplete. Field must also be required.
```

Comment:
Should this say field?

Reply:
It should not fixed here https://github.com/cerner/terra-core/pull/2670/commits/92f5fdd88a041f3c62c52a6925ccdc98d39d1c13